samples/AdministrationApi/AdministrationApi.csproj

```diff
@@ -1,6 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
```

Comment:
I am not a fan of this directive at all.
The implicit can be the tree that hides the forest. The implicit induces a form of imprecision that I don't like, but it's not my project :)

Reply:
Is your comment related to Administration API project only, or it judges all projects of the solution? :wink:

Reply:
Truly speaking, I don't like implicit usings too. But this is the way to achive the goals which are described in linked issue.
The main goal is decreasing number of lines in usings-block.

I've introduced implicit usings vs global usings. And I've got an idea to reuse global usings to decrease number of lines. So, and it seems, we can disable implicit usings at all by moving common statements to **Usings.cs** file.
What about this approach?

Reply:
Reducing the number of lines is a good thing, but you have to strike a balance.
My struggle is that every bit of code that a developer new to a project has no doubts about that code, how it works and who it depends on.
If it is necessary to investigate in several files to be sure and certain, many will not do it, and will end up doing stupid things.

Reply:
Got it! the **ImplicitUsings** property is evil! 👍 

---
> Reducing the number of lines is a good thing, but you have to strike a balance.

And this balance will be global usings in **Usings.cs** file. Did we make a deal?

---
< My struggle is that every bit of code that **a developer new to a project has no doubts about that code**, **how it works and who it depends on**.

I believe global usings will show dependencies because all using statements in one place (**Usings.cs** file). Additionally I can suggest make short comment line `// See global usings in ../Usings.cs`. But I'm not sure, because every junior C# developer knows that global usings are located in some files. But, having all global usings in one file would be a good practice.

---
> If it is necessary to investigate in several files to be sure and certain, many will not do it, and will end up doing stupid things.

I didn't get you! What to investigate to? It seems we are in the process of finding best approaches and best practices.
What are the stupid things? Juggling of usings-statements? What I know is that separating usings by multiple new lines in usings-block is a kind of stupid thing.
An example of not stupid thing is welcome!

----
> For me, "implicit" stuff, like this attribute, can hide so much. I fight every day so that my team does not reference wrongly and through useless (see dangerous) packages. With this attribute, it could escape me.

I like your care about dev team. And I will appreciate your final proposals to fix by me and get an approval from you.
- Disable implicit usings. **+**
- What else?

Will it be a kind of compromise for you if I uncover implicit `using` statements as global ones?
Will they be still implicit locating in Usings.cs file?